test/functional/rpc_psbt.py

```diff
@@ -158,56 +195,85 @@ def run_test(self):
         self.nodes[0].decodepsbt(new_psbt)
 
         # Create outputs to nodes 1 and 2
-        node1_addr = self.nodes[1].getnewaddress()
-        node2_addr = self.nodes[2].getnewaddress()
-        txid1 = self.nodes[0].sendtoaddress(node1_addr, 13)
-        txid2 = self.nodes[0].sendtoaddress(node2_addr, 13)
-        blockhash = self.nodes[0].generate(6)[0]
-        self.sync_all()
-        vout1 = find_output(self.nodes[1], txid1, 13, blockhash=blockhash)
-        vout2 = find_output(self.nodes[2], txid2, 13, blockhash=blockhash)
-
-        # Create a psbt spending outputs from nodes 1 and 2
-        psbt_orig = self.nodes[0].createpsbt([{"txid":txid1,  "vout":vout1}, {"txid":txid2, "vout":vout2}], {self.nodes[0].getnewaddress():25.999})
-
-        # Update psbts, should only have data for one input and not the other
-        psbt1 = self.nodes[1].walletprocesspsbt(psbt_orig)['psbt']
-        psbt1_decoded = self.nodes[0].decodepsbt(psbt1)
-        assert psbt1_decoded['inputs'][0] and not psbt1_decoded['inputs'][1]
-        psbt2 = self.nodes[2].walletprocesspsbt(psbt_orig)['psbt']
-        psbt2_decoded = self.nodes[0].decodepsbt(psbt2)
-        assert not psbt2_decoded['inputs'][0] and psbt2_decoded['inputs'][1]
+        # We do a whole song-and-dance here (instead of calling sendtoaddress) to get access to the unblinded transaction data to find our outputs
+        node1_addr = self.get_address(confidential, 1)
+        node1_unconf_addr = self.to_unconf_addr(1, node1_addr)
+        node2_addr = self.get_address(confidential, 2)
+        node2_unconf_addr = self.to_unconf_addr(2, node2_addr)
+        rt1 = self.nodes[0].createrawtransaction([], {node1_addr:13})
+        rt1 = self.nodes[0].fundrawtransaction(rt1)
+        rt1 = self.nodes[0].blindrawtransaction(rt1['hex'])
+        rt1 = self.nodes[0].signrawtransactionwithwallet(rt1)
+        txid1 = self.nodes[0].sendrawtransaction(rt1['hex'])
+        rt1 = self.nodes[0].decoderawtransaction(rt1['hex'])
+
+        rt2 = self.nodes[0].createrawtransaction([], {node2_addr:13})
+        rt2 = self.nodes[0].fundrawtransaction(rt2)
+        rt2 = self.nodes[0].blindrawtransaction(rt2['hex'])
+        rt2 = self.nodes[0].signrawtransactionwithwallet(rt2)
+        txid2 = self.nodes[0].sendrawtransaction(rt2['hex'])
+        rt2 = self.nodes[0].decoderawtransaction(rt2['hex'])
 
-        # Combine, finalize, and send the psbts
-        combined = self.nodes[0].combinepsbt([psbt1, psbt2])
-        finalized = self.nodes[0].finalizepsbt(combined)['hex']
-        self.nodes[0].sendrawtransaction(finalized)
         self.nodes[0].generate(6)
         self.sync_all()
 
+        for out in rt1['vout']:
+            if out['scriptPubKey']['type'] == "fee":
+                next
```

Comment:
not sure `next` is the thing you want? Seems to be an iterator thing. My guess is it's just a no-op but the logic is otherwise correct

Reply:
Oh, lol, I meant 'continue'. Which is called 'next'.... in Perl.

Reply:
(But actually 'pass' will work fine here, fixed.)